builtins/sql.ts

```diff
@@ -0,0 +1,21 @@
+import { Run, execute } from "tasks";
+
+export enum TransactionMode {
+  Auto = "auto",
+  ReadOnly = "readOnly",
+  ReadWrite = "readWrite",
+  None = "none",
+}
+
+export const sql = {
```

Comment:
Export as `default` so you can import as `import sql from 'airplane/builtins/sql`

```suggestion
export default {
```